packages/metering/lib/processors/billing.ts

```diff
@@ -118,3 +124,27 @@ async function trackUsage({ accountId, metric, delta }: { accountId: number; met
         report(new Error('Failed to track usage', { cause: err }), { accountId, metric, delta });
     }
 }
+
+async function notifyOnProdUsageThreshold({ accountId, environmentId }: { accountId: number; environmentId: number }): Promise<void> {
+    const threshold = 3;
+    try {
+        const environment = await environmentService.getRawById(environmentId);
+
+        if (environment && environment.name === 'prod') {
```

Comment:
Is this gonna work if environment is renamed

Reply:
prod can't be renamed